### PR TITLE
Translate 2018-03-28 news (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Could you review this? @ruby/www-ruby-lang-org-i18n-ko :pray:

Here is a list of raw news:

- [2018-03-28-buffer-under-read-unpack-cve-2018-8778](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-buffer-under-read-unpack-cve-2018-8778.md)
- [2018-03-28-http-response-splitting-in-webrick-cve-2017-17742](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-http-response-splitting-in-webrick-cve-2017-17742.md)
- [2018-03-28-large-request-dos-in-webrick-cve-2018-8777](https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2018-03-28-large-request-dos-in-webrick-cve-2018-8777.md)
- [2018-03-28-poisoned-nul-byte-dir-cve-2018-8780](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-poisoned-nul-byte-dir-cve-2018-8780.md)
- [2018-03-28-poisoned-nul-byte-unixsocket-cve-2018-8779](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-poisoned-nul-byte-unixsocket-cve-2018-8779.md)
- [2018-03-28-ruby-2-2-10-released](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-ruby-2-2-10-released.md)
- [2018-03-28-ruby-2-3-7-released](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-ruby-2-3-7-released.md)
- [2018-03-28-ruby-2-4-4-released](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-ruby-2-4-4-released.md)
- [2018-03-28-ruby-2-5-1-released](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-ruby-2-5-1-released.md)
- [2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914](https://github.com/ruby/www.ruby-lang.org/blob/5929dff03a9b1c08a0b05b5d03b780f929c5dddb/en/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md)

### CODE REVIEW

File: ko/news/_posts/2018-03-28-buffer-under-read-unpack-cve-2018-8778.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+tags: security
+lang: ko
+---
+
+루비의 `String#unpack` 메소드에서 공격자가 배열 범위 바깥을 읽을 수 있는 취약점이 발견되었습니다.
+이 취약점은 CVE 아이디 [CVE-2018-8778](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-8778)로 할당되었습니다.
+
+## 세부 내용
+
+`String#unpack` 은 인수로 포맷 식별자를 받으며, 식별자 `@`를 사용해 지정한 위치의 데이터를 읽어올 수 있습니다.
```
Comment (marocchino) [inline ko/news/_posts/2018-03-28-buffer-under-read-unpack-cve-2018-8778.md]: remove single space before 은

File: ko/news/_posts/2018-03-28-large-request-dos-in-webrick-cve-2018-8777.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+layout: news_post
+title: "CVE-2018-8777: WEBrick 커다란 요청을 통한 DoS 취약점"
+author: "usa"
+translator: "shia"
+date: 2018-03-28 14:00:00 +0000
+tags: security
+lang: ko
+---
+
+루비에 내장된 WEBrick에서 공격자가 커다란 요청을 전송하는 것으로 메모리를 소비하도록 만들어 DoS를 가능하게 하는 취약점이 발견되었습니다.
```
Comment (marocchino) [inline ko/news/_posts/2018-03-28-large-request-dos-in-webrick-cve-2018-8777.md]: DoS를 -> 서비스 거부 공격을

File: ko/news/_posts/2018-03-28-ruby-2-3-7-released.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+title: "루비 2.3.7 릴리스"
+author: "usa"
+translator: "shia"
+date: 2018-03-28 17:00:00 +0000
+lang: ko
+---
+
+루비 2.3.7이 릴리스 되었습니다.
+
+이번 릴리스는 70개의 버그 수정과 다수의 보안 수정을 포함합니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-3-7-released.md]: 70개의 버그 → 70여개의 버그

File: ko/news/_posts/2018-03-28-ruby-2-3-7-released.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+세부 내용은 아래 주제를 확인해주세요.
+
+* [CVE-2017-17742: WEBrick의 HTTP 응답 위장 취약점](/ko/news/2018/03/28/http-response-splitting-in-webrick-cve-2017-17742/)
+* [CVE-2018-8777: WEBrick 커다란 요청을 통한 DoS 취약점](/ko/news/2018/03/28/large-request-dos-in-webrick-cve-2018-8777/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-3-7-released.md]: How about: WEBrick의 커다란 요청을 통한 서비스 거부 공격 취약점

File: ko/news/_posts/2018-03-28-ruby-2-3-7-released.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* [CVE-2018-6914: Tempfile, Tmpdir 에서 디렉토리 순회를 동반하는 의도하지 않은 파일, 또는 디렉토리 생성 취약점](/ko/news/2018/03/28/unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-3-7-released.md]: - Tempfile, Tmpdir 에서 → tempfile, tmpdir에서
- 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-ruby-2-3-7-released.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+* [CVE-2018-8778: String#unpack의 범위 외 읽기 취약점](/ko/news/2018/03/28/buffer-under-read-unpack-cve-2018-8778/)
+* [CVE-2018-8779: UNIXServer, UNIXSocket 에서 NUL 문자 삽입을 통한 의도치 않은 소켓 생성 취약점](/ko/news/2018/03/28/poisoned-nul-byte-unixsocket-cve-2018-8779/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-3-7-released.md]: UNIXSocket 에서 → UNIXSocket에서

File: ko/news/_posts/2018-03-28-ruby-2-3-7-released.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+* [CVE-2018-8780: Dir에서 NUL 문자 주입을 통한 의도하지 않은 디렉토리 접근 취약점](/ko/news/2018/03/28/poisoned-nul-byte-dir-cve-2018-8780/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-3-7-released.md]: 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-ruby-2-3-7-released.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+      SHA512: fd91c8db3d3aa4cc962a62f27b4d1a71f6b5567ab836e6dbfbbb1553eb269d11e12faf9e36af6c489c33b54fd89dab99bfe81a563158b704877f0628d6f5bc5a
+
+* [https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.7.zip](https://cache.ruby-lang.org/pub/ruby/2.3/ruby-2.3.7.zip)
+
+      SIZE:   19911423 bytes
+      SHA1:   ec6870b39125238d8d57d642225592896ed3b6d9
+      SHA256: ffa42eeff928624a05dc7ad39426c855c6e9a757417f17b6fe9e54664ec91012
+      SHA512: c85255a7f43c7df2fb11be4f9aa96e2ae70a94d3e963ccff4d8c1349ad6d455d9a436812efb24c91e451e68b8f81e5d335c6d5811b2a0e945a7070c305054434
+
+## Release Comment
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-3-7-released.md]: 릴리스 코멘트

File: ko/news/_posts/2018-03-28-ruby-2-4-4-released.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+date: 2018-03-28 17:10:00 +0000
+lang: ko
+---
+
+루비 2.4.4이 릴리스 되었습니다.
+
+이번 릴리스는 몇몇 버그 수정과 보안 취약점 수정을 포함합니다.
+
+* [CVE-2017-17742: WEBrick의 HTTP 응답 위장 취약점](/ko/news/2018/03/28/http-response-splitting-in-webrick-cve-2017-17742/)
+* [CVE-2018-8777: WEBrick 커다란 요청을 통한 DoS 취약점](/ko/news/2018/03/28/large-request-dos-in-webrick-cve-2018-8777/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-4-4-released.md]: How about: WEBrick의 커다란 요청을 통한 서비스 거부 공격 취약점

File: ko/news/_posts/2018-03-28-ruby-2-4-4-released.md [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+* [CVE-2018-6914: Tempfile, Tmpdir 에서 디렉토리 순회를 동반하는 의도하지 않은 파일, 또는 디렉토리 생성 취약점](/ko/news/2018/03/28/unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-4-4-released.md]: - Tempfile, Tmpdir 에서 → tempfile, tmpdir에서
- 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-ruby-2-4-4-released.md [hunk 11]
```diff
@@ -0,0 +1,59 @@
...
+* [CVE-2018-8778: String#unpack의 범위 외 읽기 취약점](/ko/news/2018/03/28/buffer-under-read-unpack-cve-2018-8778/)
+* [CVE-2018-8779: UNIXServer, UNIXSocket 에서 NUL 문자 삽입을 통한 의도치 않은 소켓 생성 취약점](/ko/news/2018/03/28/poisoned-nul-byte-unixsocket-cve-2018-8779/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-4-4-released.md]: UNIXSocket 에서 → UNIXSocket에서

File: ko/news/_posts/2018-03-28-ruby-2-4-4-released.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+* [CVE-2018-8780: Dir에서 NUL 문자 주입을 통한 의도하지 않은 디렉토리 접근 취약점](/ko/news/2018/03/28/poisoned-nul-byte-dir-cve-2018-8780/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-4-4-released.md]: 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-ruby-2-4-4-released.md [hunk 13]
```diff
@@ -0,0 +1,59 @@
+---
+layout: news_post
+title: "루비 2.4.4 릴리스"
+author: "nagachika"
+translator: "shia"
+date: 2018-03-28 17:10:00 +0000
+lang: ko
+---
+
+루비 2.4.4이 릴리스 되었습니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-4-4-released.md]: 2.4.4이 → 2.4.4가

File: ko/news/_posts/2018-03-28-ruby-2-5-1-released.md [hunk 14]
```diff
@@ -0,0 +1,59 @@
...
+title: "루비 2.5.1 릴리스"
+author: "naruse"
+translator: "shia"
+date: 2018-03-28 17:20:00 +0000
+lang: ko
+---
+
+루비 2.5.1이 릴리스 되었습니다.
+
+이번 릴리스는 다수의 버그 수정과 보안 수정을 포함합니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-5-1-released.md]: You translated 'some' to '몇몇', but this one is '다수의'. Is this intentional?

File: ko/news/_posts/2018-03-28-ruby-2-5-1-released.md [hunk 15]
```diff
@@ -0,0 +1,59 @@
...
+
+* [CVE-2017-17742: WEBrick의 HTTP 응답 위장 취약점](/ko/news/2018/03/28/http-response-splitting-in-webrick-cve-2017-17742/)
+* [CVE-2018-8777: WEBrick 커다란 요청을 통한 DoS 취약점](/ko/news/2018/03/28/large-request-dos-in-webrick-cve-2018-8777/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-5-1-released.md]: How about: WEBrick의 커다란 요청을 통한 서비스 거부 공격 취약점

File: ko/news/_posts/2018-03-28-ruby-2-5-1-released.md [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+* [CVE-2018-6914: Tempfile, Tmpdir 에서 디렉토리 순회를 동반하는 의도하지 않은 파일, 또는 디렉토리 생성 취약점](/ko/news/2018/03/28/unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-5-1-released.md]: - Tempfile, Tmpdir 에서 → tempfile, tmpdir에서
- 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-ruby-2-5-1-released.md [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+* [CVE-2018-8778: String#unpack의 범위 외 읽기 취약점](/ko/news/2018/03/28/buffer-under-read-unpack-cve-2018-8778/)
+* [CVE-2018-8779: UNIXServer, UNIXSocket 에서 NUL 문자 삽입을 통한 의도치 않은 소켓 생성 취약점](/ko/news/2018/03/28/poisoned-nul-byte-unixsocket-cve-2018-8779/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-5-1-released.md]: UNIXSocket 에서 → UNIXSocket에서

File: ko/news/_posts/2018-03-28-ruby-2-5-1-released.md [hunk 18]
```diff
@@ -0,0 +1,59 @@
...
+* [CVE-2018-8780: Dir에서 NUL 문자 주입을 통한 의도하지 않은 디렉토리 접근 취약점](/ko/news/2018/03/28/poisoned-nul-byte-dir-cve-2018-8780/)
```
Comment (yous) [inline ko/news/_posts/2018-03-28-ruby-2-5-1-released.md]: 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 19]
```diff
@@ -0,0 +1,38 @@
+---
+layout: news_post
+title: "CVE-2018-6914: Tempfile, Tmpdir 에서 디렉토리 순회를 동반하는 의도하지 않은 파일, 또는 디렉토리 생성 취약점"
```
Comment (yous) [inline ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: - Tempfile, Tmpdir 에서 → tempfile, tmpdir에서
- 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 20]
```diff
@@ -0,0 +1,38 @@
...
+author: "usa"
+translator: "shia"
+date: 2018-03-28 14:00:00 +0000
+tags: security
+lang: ko
+---
+
+루비에 내장된 라이브러리인 tmpdir 에서 의도하지 않은 디렉터리를 생성하는 취약점이 발견되었습니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: tmpdir 에서 → tmpdir에서

File: ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 21]
```diff
@@ -0,0 +1,38 @@
...
+또한 내부에서 tmpdir 를 사용하는 루비에 내장된 tempfile 라이브러리에서도 의도하지 않은 파일 생성 취약점이 발견되었습니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: tmpdir 를 → tmpdir를

File: ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 22]
```diff
@@ -0,0 +1,38 @@
...
+이 취약점은 CVE 아이디 [CVE-2018-6914](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-6914)로 할당되었습니다.
+
+## 세부 내용
+
+tmpdir 라이브러리에 의해 추가되는 `Dir.mktmpdir` 메소드는 생성할 디렉토리명의 접두사나 접미사를 첫번째 매개변수로 받습니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 디렉토리명의 → 디렉터리명의

File: ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 23]
```diff
@@ -0,0 +1,38 @@
...
+접두사는 `"../"`와 같은 상대 경로를 포함할 수 있기 때문에 임의의 디렉토리를 대상으로 사용할 수 있습니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 디렉토리를 → 디렉터리를

File: ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+그러므로 외부 입력을 접두사로 사용하는 경우, 대상 디렉토리가 부적절한 권한을 가지고 있거나, 루비 프로세스가 부적절한 권한을 가지고 있다면 공격자는 임의의 디렉토리에 디렉토리나 파일을 생성할 수 있습니다.
```
Comment (yous) [inline ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 디렉토리 → 디렉터리

File: ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md [hunk 25]
```diff
@@ -0,0 +1,38 @@
...
+
+해당 버전을 사용하는 모든 사용자는 즉시 업그레이드하기 바랍니다.
+
+## 해당 버전
+
+* 루비 2.2 버전대: 2.2.9 이하
+* 루비 2.3 버전대: 2.3.6 이하
+* 루비 2.4 버전대: 2.4.3 이하
+* 루비 2.5 버전대: 2.5.0
```
Comment (yous) [inline ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 2.5.0 → 2.5.0 이하

Comment (marocchino) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 2.5.0에 밑이 있을수 있나요? p0인데..

Comment (yous) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 2.5.0-preview1, 2.5.0-rc1 등이 있어서 일부러 or earlier라고 쓴 것 같아요.

Comment (marocchino) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: #1761 그냥 복붙 미스 아닐까요.. -ㅅ-;;

Comment (yous) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 아 지금 보니 일본어판은 또 2.5.0이라고 나와 있네요. 여기에서부터 2.5.0 or earlier라고 해서 의도적인 건 줄 알았는데요. https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md

Comment (yous) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 이 글은 또 영문판이랑 동일하네요. 왠지 물어보는 게 빠를 것 같습니다. https://github.com/ruby/www.ruby-lang.org/blob/master/ja/news/_posts/2018-02-17-multiple-vulnerabilities-in-rubygems.md

Comment (marocchino) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 물어봤습니다.

Comment (marocchino) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: > Conclusion: you can choose to say or not to say "earlier" as you like :)

맘대로 하라고하시네요. -ㅁ-

Comment (riseshia) [reply ko/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md]: 😇 Then I vote +1 using `이하`

File: ko/news/_posts/2018-03-28-buffer-under-read-unpack-cve-2018-8778.md (hunk 1 above)
Comment (marocchino) [reply ko/news/_posts/2018-03-28-buffer-under-read-unpack-cve-2018-8778.md]: !!!

File: ko/news/_posts/2018-03-28-ruby-2-5-1-released.md (hunk 14 above)
Comment (riseshia) [reply ko/news/_posts/2018-03-28-ruby-2-5-1-released.md]: nope, It just copy & paste from previous news ;) I will change to '몇몇' for consistency.

**Comment by riseshia** [comment]
Fix commented texts da3b2ea0

**Pull request merged**